README.md

```diff
@@ -1,36 +1,59 @@
 # Utils
 
-[![Build Status](https://travis-ci.org/kubernetes/utils.svg?branch=master)](https://travis-ci.org/kubernetes/utils)
+[![Build Status]](https://travis-ci.org/kubernetes/utils)
 
-Utils is a set of golang libraries that are not specific to
-Kubernetes. They should be available and useful to any other Go project
-out there.
+A set of Go libraries that provide low-level,
+kubernetes-independent packages supplementing the [Go
+standard libs].
 
 ## Purpose
 
-The Kubernetes project uses a lot of Golang patterns that are re-used in many
-difference places, and as the project is further split into different
-repositories, this repository is the perfect place to hold this common code.
+As Kubernetes grows and spins functionality out of its
+[core] and into cooperating repositories
+like [apiserver], [kubectl], [kubeadm], etc.,
+the need arises for leaf repos to house shared code
+and avoid cycles in repo relationships.
 
-## Goals
+This repo is intended to hold shared utilities
+with no Kubernetes dependence that may be of interest
+to any Go project.  See these [instructions for moving]
+an existing package to this repo.
+  
+The [common] repo, on the other hand, holds shared utilities
+that may depend on both this repo and other near-leaf
+repos like [api].
 
-Go libraries in this repository must be:
+## Criteria for adding code here
 
-- Generic enough that they are useful for external/non-kubernetes
-  projects,
-- Well factored, well tested and reliable,
-- Be completely go compliant (go get/build/test/etc)
-- Have enough complexity to be shared,
-- Have stable APIs, or backward compatible.
+- Used by multiple Kubernetes repos.
```

Comment:
I don't know if I want this to be the first/most important criteria though. I feel like being generic is more important.